### PR TITLE
GStreamer framework incl. graphene and gentrans

### PR DESCRIPTION
**Pull request by ppp0**
see https://github.com/cargomedia/puppet-packages-ubuntu/issues/3

### CODE REVIEW

File: modules/gstreamer/manifests/plugins/base.pp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class gstreamer::plugins::base (
+  $version = '1.6.1',
+  $with_orc = true,
```
Comment (ppp0) [inline modules/gstreamer/manifests/plugins/base.pp]: @kris-lab not sure if we should include orc like this, as option to plugins::base? wdyt?

**Comment by ppp0** [comment]
@kris-lab you take over? with @dergraf ? thank you!

**Comment by ppp0** [comment]
@tomaszdurka please review

File: modules/gstreamer/manifests/plugins/gentrans.pp [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class gstreamer::plugins::gentrans (
+  $version = 'latest',
+) {
+
+  require 'apt::source::cargomedia'
+  require 'gstreamer::plugins::base'
+
+  package { 'gst-entrans':
```
Comment (tomaszdurka) [inline modules/gstreamer/manifests/plugins/gentrans.pp]: Is this entrans or gentrans? Package doesn't match class name.

File: modules/gstreamer/manifests/plugins/libav.pp [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class gstreamer::plugins::libav (
+  $version = 'latest',
+) {
+
+  require 'apt::source::cargomedia'
+  require 'gstreamer::plugins::base'
+
+  package { 'gstreamer1.0-libav':
```
Comment (tomaszdurka) [inline modules/gstreamer/manifests/plugins/libav.pp]: Class is plugin::, but package not?

File: modules/gstreamer/manifests/plugins/base.pp [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class gstreamer::plugins::base (
+  $version = 'latest',
+) {
+
+  require 'apt::source::cargomedia'
+
+  package { 'gstreamer1.0-plugins-base':
```
Comment (tomaszdurka) [inline modules/gstreamer/manifests/plugins/base.pp]: Should base require also gstreamer class ?

File: modules/gstreamer/manifests/plugins/libav.pp (hunk 3 above)
Comment (ppp0) [inline modules/gstreamer/manifests/plugins/libav.pp]: ah, these misteries of package naming...

package: http://packages.ubuntu.com/vivid/gstreamer1.0-libav

but in fact, this is a proper plugin to gstreamer (see description)

File: modules/gstreamer/manifests/plugins/gentrans.pp (hunk 2 above)
Comment (ppp0) [inline modules/gstreamer/manifests/plugins/gentrans.pp]: It's not my fault :smile: ... 

see http://gentrans.sourceforge.net/

File: modules/gstreamer/manifests/plugins/base.pp (hunk 4 above)
Comment (ppp0) [inline modules/gstreamer/manifests/plugins/base.pp]: There is no `gstreamer` as such, we use the tools for some useful binaries, but that's it

**Comment by ppp0** [comment]
@tomaszdurka please review

**Comment by tomaszdurka** [comment]
Looks good, pair-finished with @kris-lab

**Pull request merged**